aptos-move/move-examples/token_objects/guild/sources/guild.move

```diff
@@ -0,0 +1,309 @@
+/// This module implements the guild token (non-fungible token) including the
+/// functions create the collection and the guild tokens.
```

Comment:
Can you add more details about the flow and what are the different account roles here?

Reply:
Done.